video_xblock/tests/unit/mocks/wistia.py

```diff
@@ -38,208 +32,3 @@ def get(self):
         if self.event == 'not_authorized':
             self.return_value = ResponseStub(status_code=401)
         return lambda x: self.return_value
-
-
-class WistiaDefaultTranscriptsMock(BaseMock):
```

Comment:
That group mocks approach violates basic principles of unit testing:
- too much effort to figure out what happens - it's hard to maintain;
- every single method's implementation though has a unified interface still uses its own utils which should be mocked;
- 'keep unit tests dumb' (c) ...